src/js/tabs/osd.js

```diff
@@ -581,7 +582,7 @@ OSD.constants = {
             draw_order: 160,
             positionable: true,
             preview: function (osd_data) {
-                return ' 399.7' + FONT.symbol(osd_data.unit_mode === 0 ? SYM.FEET : SYM.METRE);
+                return FONT.symbol(SYM.ALTITUDE) + '399.7' + FONT.symbol(osd_data.unit_mode === 0 ? SYM.FEET : SYM.METRE);
```

Comment:
I think you forgot one space, between the symbol and the number.

Reply:
You're right, the symbol is prepended to 5 positions of the number plus the dot. Fixed.